### PR TITLE
[Fix](bangc-ops): add scale limitation in mlu_op.h

### PR DESCRIPTION
**Pull request by PetrelYy**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

modified:   bangc-ops/mlu_op.h

### CODE REVIEW

**Review by mahxn0 — APPROVED** [review]

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -11569,42 +11573,42 @@ mluOpDeformRoiPoolBackward(const mluOpHandle_t handle,
  * Pointer to the MLU memory that stores the output tensor. The shape of
  * argmax_idx is [N, H * W, 4, C].
  * @param[in] argmax_idx_desc
- * Descriptor of \b argmax_idx, containing dimension and the layout of \b argmax_idx.
+ * Descriptor of \b argmax_idx, containing dimension and the layout of \b argmax_idx .
```
Comment (Unireverse) [inline bangc-ops/mlu_op.h:11576]: 多了空格

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -11569,42 +11573,42 @@ mluOpDeformRoiPoolBackward(const mluOpHandle_t handle,
...
+ * indices of maximum values after the max pooling. The shape of \b argmax_idx
+ * is [N, H * W, 4, C].
  *
  * @par Return
  * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM, ::MLUOP_STATUS_NOT_SUPPORTED,
  *   ::MLUOP_STATUS_EXECUTION_FAILED
  *
- * @par Formula
- * - See "BorderAlignForward Operator" section in "Cambricon BANG C OPS User Guide" for details.
- *
```
Comment (Unireverse) [inline bangc-ops/mlu_op.h:11583]: 这个为啥删掉

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -11569,42 +11573,42 @@ mluOpDeformRoiPoolBackward(const mluOpHandle_t handle,
...
- * - This function supports the following data types for input tensor \b input, \b boxes, \b pool_size,
- *    \b output and \b argmax_idx. Data type of the \b input, \b boxes, and \b output tensors must be the same.
- *   - input tensor: half, float.
- *   - boxes tensor: half, float.
- *   - pool_size: int32_t.
- *   - output tensor: half, float.
- *   - argmax_idx: int32_t.
+ * - The supported data types of input and output tensors are as follows:
+ *   <b>Note that the data type of \b input , \b boxes , and \b output
+ *   must be the same.
```
Comment (Unireverse) [inline bangc-ops/mlu_op.h:11589]: 这个note感觉放到类型枚举后面比较好

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -11685,82 +11689,74 @@ mluOpBorderAlignForward(mluOpHandle_t handle,
...
- * boxes, \b argmax_idx, \b pool_size and \b grad_input. Data types of the \b grad_output, \b boxes,
- * and \b grad_input tensors should be the same.
- *  - grad_output tensor: half, float.
- *  - boxes tensor: half, float.
- *  - argmax_idx: int32_t.
- *  - pool_size: int32_t.
- *  - grad_input tensor: half, float.
+ * - The supported data types of input and output tensors are as follows:
+ *   <b>Note that the data type of \b grad_output , \b boxes , and \b grad_input
+ *   must be the same.
```
Comment (Unireverse) [inline bangc-ops/mlu_op.h:11734]: 同上

File: bangc-ops/mlu_op.h (hunk 3 above)
Comment (PetrelYy) [reply bangc-ops/mlu_op.h]: 已修改

File: bangc-ops/mlu_op.h (hunk 4 above)
Comment (PetrelYy) [reply bangc-ops/mlu_op.h]: 已修改

**Review by Unireverse — APPROVED** [review]

**Pull request merged**